### PR TITLE
First_Blog_What_Programming_Language_To_Choose_As_Beginner

### PR DESCRIPTION
**Pull request by pulkitgovrani**
Hello team,
Please review the blog.

### CODE REVIEW

File: doc/source/What Programming Language to choose as a beginner.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## What Programming Language to choose as a beginner?
```
Comment (basalumutgazi) [inline doc/source/What Programming Language to choose as a beginner.md:1]: Main title should start with only one hashtag (#)

File: doc/source/What Programming Language to choose as a beginner.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+This is how we can print “Hello World” in Python:
+
+![Python](_static/images/What_Programming_Language_Images/python.png)
+
+So I am pretty sure you must have made your mind till now that learning python is both easy and beneficial for you.
+
+## About the Author
+
+
```
Comment (basalumutgazi) [inline doc/source/What Programming Language to choose as a beginner.md:27]: Too many empty lines

**Review by basalumutgazi — CHANGES_REQUESTED** [review]
Please see my comments.
Also note that all the filenames should be all lowercase and without spaces. 
e.g.: what-programming-language-to-choose-as-beginner.md

File: doc/source/What Programming Language to choose as a beginner.md (hunk 1 above)
Comment (basalumutgazi) [reply doc/source/What Programming Language to choose as a beginner.md:1]: Also you will need an emoji for the title

Comment (pulkitgovrani) [reply doc/source/What Programming Language to choose as a beginner.md]: Thanks for reviewing,
I have done the changes. Please check again.

**Pull request closed without merging**